.github/main.workflow

```diff
@@ -0,0 +1,8 @@
+workflow "Automatic Rebase" {
+  on = "issue_comment"
+  resolves = ["Rebase"]
+}
+
+action "Rebase" {
+  uses = "cirrus-actions/rebase@latest"
```

Comment:
Why is the `secrets = ["GITHUB_TOKEN"]` entry from the documentation not required?

Reply:
HTB, I have no idea. I've created the file from scratch using the UI editor, not manually from the example. My guess is that the example is simply outdated. But after all, I'm not yet sure that the workflow will actually work. I wanted to verify this after merging 😉

Reply:
@mnonnenmacher, I've tested it at https://github.com/sschuberth/actions-playground/actions and confirmed that the (literal string) `GITHUB_TOKEN` is indeed required. I'm also using the image from DockerHub now instead of building it from sources.

Reply:
Thanks for testing.